### DISCUSSION TITLE
Releases (link, now tab) doesn't show on repositories without any release yet

### DISCUSSION THREAD

**Discussion by vhanla** [community/community#6831] [Other Feature Feedback, Questions, & Ideas] [answered]
I created a new repository and the Releases tab/link doesn't show, I had to manually add "/releases" to the repo URL path, hit enter and accesed the create a new release draft page. 

![imagen](https://user-images.githubusercontent.com/1015823/139118518-f16f5252-d6ab-40e3-90bc-0be5154ab8dd.png)

**Comment by Katsute** [comment, marked as answer]
The releases tab is on the right sidebar. You can press the gear ⚙ next to the **About** to turn it on.

![releases](https://user-images.githubusercontent.com/58778985/139119390-28d5625e-aecb-4ab1-82e9-74395cb2fd1f.PNG)
